### DISCUSSION TITLE
v4 版本   c++ 推理直接用代码里的 deploy/cpp_infer/  这个可用吗，需不需要改那里？

### DISCUSSION THREAD

**Discussion by huangguifeng** [PaddlePaddle/PaddleOCR#13283] [Q&A]
v4 版本   c++ 推理直接用代码里的 deploy/cpp_infer/  这个可用吗，需不需要改那里？ deploy/cpp_infer/   是否支持v4 版本

**Comment by UserWangZz** [comment]
目前deploy中的代码已经不维护了，建议使用FastDeploy仓库进行模型部署

**Comment by Phymin** [comment]
既然不维护了，但我看cpp_infer却一直在更新，反而FastDeploy仓库已经很久没更新了，这是什么原因？
